burbank-ca.md

```diff
@@ -0,0 +1,6 @@
+## Burbank, CA
+
+### Great places to eat
```

Comment:
I hang out in Burbank all the time as I went to high school in Panorama City! What's your favorite place to eat?